gdi/opentelemetry/install-windows.rst

```diff
@@ -112,8 +112,14 @@ For example, ``https://ingest.<SPLUNK_REALM>.signalfx.com/v1/log``.
 
 To configure the package to send log events to a custom HTTP Event Collector (HEC) endpoint URL, you can specify the following parameters for the installer script:
 
-* ``SPLUNK_HEC_URL = "<URL>"``
-* ``SPLUNK_HEC_TOKEN = "<TOKEN>"``
+* ``hec_url = "<SPLUNK_HEC_URL>"``
+* ``hec_token = "<SPLUNK_HEC_TOKEN>"``
+
+For example:
+
+.. code-block:: PowerShell
+
+  & {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/splunk-otel-collector.ps1')); $params = @{access_token = "SPLUNK_ACCESS_TOKEN"; realm = "SPLUNK_REALM"; hec_url = "SPLUNK_HEC_URL"; hec_token = "SPLUNK_HEC_TOKEN"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
```

Comment:
This update to the example is definitely better than the existing content!

Are you certain that customers know how to find the `<SPLUNK_HEC_URL>` and `<SPLUNK_HEC_TOKEN>`? Docs team often gets feedback from customers that the instructions don't tell them how to find the values they have to supply, or what the format of the values is. 

Also, in your PowerShell example, I suggest you use `hec_url = "<SPLUNK_HEC_URL>"; hec_token = "<SPLUNK_HEC_TOKEN>"`. I assume that PowerShell throw an error if customers don't replace the variable with the actual value, because the angle brackets won't parse. I intend that behavior; customers will have clear notice that they have to replace the string with its actual value.

Reply:
@jmalin-splunk, we're working on more updates separately. 

@mfoulds-splunk, this is a quick fix that needs to go live, so I'm merging.